forge-gui/res/languages/en-US.properties

```diff
@@ -2948,3 +2948,4 @@ lblEffect=Effect
 lblEmblem=Emblem
 lblBoon=Boon
 lblExitToWoldMap=Exit to the World Map?
+lblWouldYouLikeDestroy=Would you like to destroy {0} ?
```

Comment:
you should add it to all languages

Reply:
You mean putting the english through google translate ?

Reply:
no, just mirror it so translators can see what's missing easily without diff